crates/bevy_pbr/src/render_graph/pbr_pipeline/pbr.frag

```diff
@@ -38,10 +38,8 @@ const int MAX_LIGHTS = 10;
 
 struct Light {
     mat4 proj;
-    vec3 pos;
```

Comment:
It seems this change may be reverted (memory layout is the same). I've tested that in bevy_webgl2 and it seems to work.

Reply:
yes, in `std140` memory layout is the same between `vec3` and `vec4`. That's why, from what I read, `vec3` should be avoided to not make mistake when reading the code to think that the layout is different